test-mirage/test.ml

```diff
@@ -4,19 +4,14 @@ open Capnp_rpc_net
 open Examples
 
 module Time = struct
-  type 'a io = 'a Lwt.t
   let sleep_ns ns = Lwt_unix.sleep (Duration.to_f ns)
 
-  type t = unit
-
   let period_ns () = None
   let elapsed_ns () = 0L
-  let disconnect () = Lwt.return_unit
 end
 
 module Random = struct
   type g = unit
-  type buffer = Cstruct.t
 
   let generate ?g n = ignore g; Cstruct.create n
```

Comment:
I don't know much about this code, but why do you mock the `Random` implementation? I'd suggest to use a normal random, or if you depend on it always to return zero to use `Mirage_crypto_rng.Null` (which behaves differently: it needs to be seeded with a big enough Cstruct.t that is emitted in chunks on calls to generate)